### PR TITLE
add stefenphelps.com to showcase

### PR DESCRIPTION
**Pull request by stefenphelps**

### CODE REVIEW

File: src/data/showcase/elian-codes.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"title": "Elian Codes",
```
Comment (tony-sull) [inline src/data/showcase/elian-codes.json:2]: Revert this file and we should be all set 👍

**Review by tony-sull — CHANGES_REQUESTED** [review]
Thanks for opening this PR!  Added one comment to revert an unrelated file and we'll get this merged in 🎉

File: src/data/showcase/elian-codes.json (hunk 1 above)
Comment (stefenphelps) [reply src/data/showcase/elian-codes.json:2]: ack! Will do. Thanks!

**Comment by tony-sull** [comment]
Sorry for the delay here!  The Netlify deploys went a bit haywire and I wanted to double check why they were hanging before the merge :+1:

**Pull request merged**